format/spec.md

```diff
@@ -513,6 +514,17 @@ Manifests for a snapshot are tracked by a manifest list.
 
 Valid snapshots are stored as a list in table metadata. For serialization, see Appendix C.
 
+Statistics files' metadata within `statistics` field is a struct with the following fields:
```

Comment:
I think this should be a sub-section, like `Manifest Lists` below. It should also be located after the `Snapshot Retention Policy` sub-section. There's no need to place this before manifest lists.

In the new section for `Statistics file`, I think this should add any disclaimers about the use of stats, like @danielcweeks suggested.

Reply:
moved & added